README.md

```diff
@@ -91,7 +91,7 @@ For many particle physics experiments, a lot of data is stored within ROOT files
 
 | Package name | Use | Pro | Con | Further information |
 | ------------ | --- | --- | --- | ------------------- |
-| [uproot](https://github.com/scikit-hep/uproot) | Native Python ROOT I/O | Easy to install, fast, no dependencies | Purely for reading ROOT files to exit ROOT ecosystem | | 
+| [uproot](https://github.com/scikit-hep/uproot) | Native Python ROOT I/O | Easy to install, fast, no dependencies | Can only write ROOT files with specific objects | | 
```

Comment:
You should not remove the info that the package reads ROOT files. You do want to add that it can also write ROOT files with specific objects.